test/acceptance/features/bindAppToService.feature

```diff
@@ -7,183 +7,256 @@ Feature: Bind an application to a service
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
-        * PostgreSQL DB operator is installed
+        * OLM Operator "backend" is running
+        * The Custom Resource Definition is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backends.stable.example.com
+                annotations:
+                    service.binding: path={.status.data.dbCredentials},objectType=Secret,elementType=map
+            spec:
+                group: stable.example.com
+                versions:
+                  - name: v1
+                    served: true
+                    storage: true
+                scope: Namespaced
+                names:
+                    plural: backends
+                    singular: backend
+                    kind: Backend
+                    shortNames:
+                      - bk
+            """
+        * The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: backend-secret
+            stringData:
+                username: AzureDiamond
+                password: hunter2
+            """
 
     @smoke
-    Scenario: Bind an imported Node.js application to PostgreSQL database in the following order: Application, DB and Service Binding
-        Given Imported Nodejs application "nodejs-rest-http-crud-a-d-s" is running
-        * DB "db-demo-a-d-s" is running
+    Scenario: Bind an application to backend service in the following order: Application, backend service and Service Binding
+        Given Generic test application "gen-app-a-b-s" is running
+        And The Custom Resource is present
+            """
+            apiVersion: stable.example.com/v1
+            kind: Backend
+            metadata:
+                name: service-a-b-s
+            status:
+                data:
+                    dbCredentials: backend-secret
+            """
         When Service Binding is applied
             """
             apiVersion: binding.operators.coreos.com/v1alpha1
             kind: ServiceBinding
             metadata:
-                name: binding-request-a-d-s
+                name: service-binding-a-b-s
```

Comment:
is this change really needed in this PR?

Reply:
`a-d-s` stands for `app-databse-servicebinding` but now I replaced databse with `backend-service` so I changed it to `a-b-s`.

Reply:
In the examples have permutations of App, DB and ServiceBinding - therefore `a-d-s`. However, here we have App, Service and the Binding, so what about using `s` instead of `d` and `b` instead of `s` so we would have permutations of `a-s-b`? 

I like the idea of naming things properly, consistently and uniquely to avoid collisions.